### ISSUE TITLE
wrong destination directory

### ISSUE THREAD

**Issue by drakon660** [kerryjiang/SuperSocket.FtpServer#2]
Hi
When I use winscp to log into ftp server i get "E:\Sftp-a" was not found. My destination is actually E:\Sftp but what "-a" doing here?

best regards
Chris

**Comment by kerryjiang** [comment]
winscp? did you try ftp client?

**Comment by drakon660** [comment]
Yes I did, and it is working, with filezilla client it is working too but winscp it is not working.
Cloud you test it against winscp? Where is the problem? Is it different ftp implementation?

**Comment by kerryjiang** [comment]
Do you have a screenshot or a log of the ftp command interaction?

**Comment by drakon660** [comment]
Here you are.
![err](https://cloud.githubusercontent.com/assets/3002778/6288342/91972fb8-b918-11e4-98c6-e4ade407c78f.png)

**Comment by kerryjiang** [comment]
I meant the information you can find from the client side.

**Comment by drakon660** [comment]
![e1](https://cloud.githubusercontent.com/assets/3002778/6289949/003fba92-b924-11e4-9b23-859c6852d3e5.png)

**Comment by evgenekov** [comment]
![default](https://cloud.githubusercontent.com/assets/12069644/20688930/fe5d46f0-b5d3-11e6-9206-843c119e8fc6.png)

Set Off to Support for listing of hidden files
